[16.0][MIG] l10n_br _fiscal

mileo opened the pull request:
- [x] Adapt field_view_get to get_view;
- [x] Adapt menus to a two level only structure;
- [x] Fix tests;

mileo committed:
[MIG][l10n_br_fiscal][REF] Security rule

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

rvalyi committed:
[MIG] manual migration to 13.0

rvalyi committed:
[MIG] l10n_br_fiscal: Migration to 14.0

marcelsavegnago committed:
[IMP] l10n_br_fiscal: tax.py - remove discount from tax base value

renatonlima committed:
[REF] _compute_tax_base to implements add_to_base and remove_from_base

mileo committed:
[REF] Operation view usability

mileo committed:
[REF] Edoc Purpouse on Operation

mileo committed:
[IMP] Track visibility on fiscal operation

mileo committed:
[IMP] Fiscal tax definition name and related fields

mileo committed:
[IMP] Operation and Operation Line usability

mileo committed:
[NEW] Tax definition inverse relation

mileo committed:
[NEW] CST code to tax definiton tree

rvalyi committed:
temporary hack to get tests pass

rvalyi committed:
[MIG] manual migration to 14.0

marcelsavegnago committed:
[FIX] pre-commit

rvalyi committed:
[FIX] clean related fields.Selection

rvalyi committed:
[ADD] add missing access righs

rvalyi committed:
[REM] remove useless mixin doc view

rvalyi committed:
[MIG] taxes of new lines are NewId records on v13

rvalyi committed:
pre-commit

renatonlima committed:
[REF] add fiscal document key validation

renatonlima committed:
[FIX] fiscal document demo data

felipemotter committed:
[REF] Delete payment model from fiscal

mbcosta committed:
[REF] Inform Product Fiscal Type for Sales Devolution Fiscal Operation.

mbcosta committed:
[IMP] Inform Product Fiscal Type for Purchase and Purchase Devolution Fiscal Operations.

renatonlima committed:
[ADD] script migration for l10n_br_fiscal.document.related

renatonlima committed:
pre-commit

renatonlima committed:
[FIX] imports

renatonlima committed:
[REF] fiscal document line must be store=True

rvalyi committed:
[REF] @api.depends for amount fields store=True

renatonlima committed:
[FIX] Document related key constrain

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

rvalyi committed:
[MIG] l10n_br_fiscal: Migration to 13.0

rvalyi committed:
[FIX] no digits keyword for Monetary fields

rvalyi committed:
[MIG] remove bad size keywords

rvalyi committed:
[MIG] track_visibility=onchange -> tracking=True

rvalyi committed:
[FIX] readony -> readonly

rvalyi committed:
[REF] regenerated l10n_br_fiscal/i18n/l10n_br_fiscal.pot

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
[UPD] README.rst

rvalyi committed:
[FIX] no tracking=True in abstract model

rvalyi committed:
[REF] fake certificate triggers DeprecationWarning

detail: erpbrasil/assinatura/certificado.py:30: DeprecationWarning: str for passphrase is no longer accepted
this was creating a WARNING on RUNBOT.

antoniospneto committed:
[MIG] fix for_xml_id

antoniospneto committed:
[MIG] fix call render_template

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
[UPD] README.rst

rvalyi committed:
[MIG][REF] refactor fiscal fields injection

OCA-git-bot committed:
l10n_br_fiscal 14.0.2.0.0

renatonlima committed:
[FIX] postprocess_and_fields args

OCA-git-bot committed:
l10n_br_fiscal 14.0.2.1.0

renatonlima committed:
[FIX] _search_name method in data.abstract

renatonlima committed:
[FIX] wrong attrs fields attribute

renatonlima committed:
Update TIPI, II

OCA-git-bot committed:
l10n_br_fiscal 14.0.2.2.0

renatonlima committed:
[FIX] _update_fiscal_quantity method

renatonlima committed:
[FIX] test estimate taxes

renatonlima committed:
[FIX] fiscal document _compute_amount depends

renatonlima committed:
[FIX] Change fields.Float to fields.Monetary

renatonlima committed:
[ADD] fiscal document mixin amount for ii and icms difal taxes

renatonlima committed:
[ADD] Fiscal CEST demo data

renatonlima committed:
[FIX] mapping IPI guideline tax code

renatonlima committed:
Set amount fields as store=True

rvalyi committed:
[FIX] always assign financial_discount_value

rvalyi committed:
[FIX] store field typo

rvalyi committed:
[FIX] fix inconsistent 'compute_sudo'

OCA-git-bot committed:
l10n_br_fiscal 14.0.3.0.0

renatonlima committed:
Added some index in key fields

OCA-git-bot committed:
l10n_br_fiscal 14.0.3.1.0

OCA-git-bot committed:
l10n_br_fiscal 14.0.3.2.0

renatonlima committed:
Rename line_ids to fiscal_line_ids

renatonlima committed:
Rename line_ids to fiscal_line_ids in tests

rvalyi committed:
[REM] remove dead code from before the v12 migration

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.0.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

antoniospneto committed:
[FIX] default fiscal dummy by company

mbcosta committed:
[FIX] l10_br_fiscal: Get the current company from context should be made with self.env.company .

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.0.1

renatonlima committed:
Update fiscal data CFOP

renatonlima committed:
Removed ind_final from Fiscal Operation

renatonlima committed:
Added field ind_final in Partner

renatonlima committed:
Adapt compute_taxes method

renatonlima committed:
Convert icms_origin to property in Product Template

antoniospneto committed:
[FIX] document confirmation when the issuer is the partner

renatonlima committed:
Removed ind_final from fiscal operation onchange

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.1.0

antoniospneto committed:
[IMP] Minimal structure for XML validation

antoniospneto committed:
[FIX] remove requerid true for fiscal_dummy_id

renatonlima committed:
Convert fiscal_type to property in Product Template

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.2.0

renatonlima committed:
[FIX] Só permitir editar o campo edoc_purpose em draft e corrigido label

renatonlima committed:
pre-commit

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.3.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.4.0

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.5.0

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.6.0

marcelsavegnago committed:
[FIX] l10n_br_fiscal: remove widgets from chatter section

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.6.1

antoniospneto committed:
[FIX] properties in fiscal products demo data

renatonlima committed:
[FIX] wrong tracking parameter in res.partner ind_final field

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.7.0

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.8.0

rvalyi committed:
pylint import fix

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.9.0

marcelsavegnago committed:
[FIX] l10n_br_fiscal: fix view access permission

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.9.1

antoniospneto committed:
[FIX] compute certificate name

antoniospneto committed:
[REF] Remove pyOpenSLL dependence

antoniospneto committed:
[REF] ported the creation of the certificate to erpbrasil.assinatura

antoniospneto committed:
[FIX] enable commented certificate fake code

rvalyi committed:
erpbrasil.assinatura -> erpbrasil.assinatura-nopyopenssl

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.10.0

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.11.0

mbcosta committed:
[REF] l10n_br_fiscal: Moved Comments creation method to be also used when l10n_br_account are not installed.

OCA-git-bot committed:
[IMP] update dotfiles [ci skip]

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.12.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.13.0

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.14.0

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.15.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

marcelsavegnago committed:
[FIX] l10n_br_fiscal: force pdf regeneration

marcelsavegnago committed:
[IMP] l10n_br_fiscal: add cnae_id on document_fiscal_line_mixin

marcelsavegnago committed:
[IMP] l10n_br_fiscal: add cnae_id on document_fiscal_line_view

marcelsavegnago committed:
[FIX] l10n_br_fiscal: fix tracking parameter

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.16.0

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.17.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

marcelsavegnago committed:
[IMP] l10n_br_fiscal: add city_taxation_code_ids on tax_definition

renatonlima committed:
tax_icms_or_issqn visible in fiscal document lines

OCA-git-bot committed:
l10n_br_fiscal 14.0.4.17.1

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.0.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

renatonlima committed:
Update python lib erpbrasil.assinatura version 1.4.0

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.1.0

renatonlima committed:
[REF] API request timeout and check if IBPT Service Unavailable

renatonlima committed:
[FIX] Check IBPT API result is None

renatonlima committed:
[REF] IBPT Tests

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.1.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

hirokibastos committed:
[FIX] Fixed icms regulation using partner's state_id to get origin for ICMS Difal

renatonlima committed:
[FIX] l10n_br_fiscal: create fiscal dummy with document number null

marcelsavegnago committed:
[FIX] l10n_br_fiscal: force save tax_icms_or_issqn

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.2.0

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 89.6% (1234 of 1377 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/pt_BR/

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.3.0

renatonlima committed:
[FIX] check_number_in_use method

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.4.0

marcelsavegnago committed:
[FIX] l10n_br_fiscal: change ibpt service timeout value

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.4.1

renatonlima committed:
[REF] Convert edoc_purpose selection in fiscal constant

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.4.2

antoniospneto committed:
[FIX] Correct legal_name for Prefeitura Municipal partner

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.4.3

antoniospneto committed:
[FIX] l10n_br_fiscal: onchange ind_final

OCA-git-bot committed:
l10n_br_fiscal 14.0.5.4.4

antoniospneto committed:
[IMP] l10n_br_fiscal: add verification for freight, insurance and others values

antoniospneto committed:
[IMP] add ind_final option to map ICMS

Co-authored-by: Felipe <felipe@engenere.one>

antoniospneto committed:
[IMP] l10n_br_fiscal: tests for ICMS Regulation

rvalyi committed:
[IMP] contributors + some readme love

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.6.0.0

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

renatonlima committed:
[FIX] map_tax_icms_difal must get icms tax defs from destination

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.0.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.0.1

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

rvalyi committed:
[FIX] CONTRIBUTORS

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

antoniospneto committed:
[FIX] l10n_br_fiscal: load certificate data

antoniospneto committed:
[FIX] l10n_br_fiscal: next fiscal number

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.1.1

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.1.2

renatonlima committed:
[FIX] Allows create line_section and line_note

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.2.0

felipemotter committed:
[FIX] pacote UOM category

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.3.0

antoniospneto committed:
[IMP] add check company for document fiscal

antoniospneto committed:
[FIX] dont set None in float

marcelsavegnago committed:
[IMP] l10n_br_fiscal: add brazilian fields on partner _commercial_fields

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.4.0

felipemotter committed:
[FIX] fiscal dummy in new companies

felipemotter committed:
[ADD] test company consistency in fiscal dummy

hirokibastos committed:
[FIX] Removed partner.is_company from difal conditions

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.4.1

renatonlima committed:
[FIX] add_to_base def

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.4.2

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.5.0

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 91.3% (1261 of 1381 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/pt_BR/

renatonlima committed:
Add ICMS DIFAL Amounts in view

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.6.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

augustodinizl committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 91.3% (1262 of 1381 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/pt_BR/

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 91.4% (1263 of 1381 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/pt_BR/

mileo committed:
[ADD] NFC-e constant [MOV] FISCAL_PAYMENT_MODE

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.6.1

antoniospneto committed:
[FIX] fiscal: security restriction for create fiscal dummy

OCA-git-bot committed:
l10n_br_fiscal 14.0.7.6.2

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 92.2% (1274 of 1381 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/pt_BR/

mbcosta committed:
[IMP] l10n_br_fiscal: Included parameter to define if the values of Freight, Insurance and Other Costs should be inform By Line or By Total.

Commit:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.8.0.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

marcelsavegnago committed:
[IMP] l10n_br_fiscal: add force_compute_delivery_costs_by_total

antoniospneto committed:
[FIX] l10n_br_fiscal: change deleted product demo

OCA-git-bot committed:
l10n_br_fiscal 14.0.8.0.1

mileo committed:
[REF] attribute-string-redundant l10n_br_fiscal

mileo committed:
[REF] xml-deprecated-tree-attribute l10n_br_fiscal

mileo committed:
[REF] str-format-used l10n_br_fiscal

Commit:
[UPD] Update l10n_br_fiscal.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

mileo committed:
fixup! [REF] attribute-string-redundant l10n_br_fiscal

Commit:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 14.0.8.0.2

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_fiscal/

mileo committed:
[REF] missing-return l10n_br_fiscal

OCA-git-bot committed:
l10n_br_fiscal 14.0.8.0.3

mileo committed:
[REF] raise-missing-from l10n_br_fiscal

OCA-git-bot committed:
l10n_br_fiscal 14.0.8.0.4

mileo committed:
[MIG][l10n_br_fiscal] bump module version to 16

mileo committed:
[FIX] 16.0 pyupgrade auto format

mileo committed:
[IMP] l10n_br_fiscal pre-commit stuff

mileo committed:
[REF] 16.0 pylint missing-files

mileo committed:
[FIX] Fiscal Document View is restricted to the group(s) base.group_multi_company

mileo committed:
[FIX] Document Serie View is restricted to the group(s) base.group_multi_company

mileo committed:
[FIX] base_address_city -> base_address_extended l10n_br_fiscal

mileo committed:
[MIG] email template to v16 syntax

mileo committed:
[MIG] icms regulation fields_view_get -> get_view

mileo commented:
/ocabot migration

mileo commented:
/ocabot migration l10n_br_fiscal

mileo commented:
/ocabot migration l10n_br_fiscal

mileo commented:
Pessoal na v16 pelo que eu vi não será possível mantermos essa estrutura de três níveis de menu:

![image](https://user-images.githubusercontent.com/1975978/197440218-730f6025-79b4-4e23-9a72-21b0db225b8b.png)

Só esta sendo possível usar 2 níveis, se alguém tiver alguma sugestão de como organizar esses menus por favor comenta aqui.

rvalyi commented:
eu imagino que isso não vai pegar apenas com a gente (imagina algum menu tipo de SPED com uma limitação dessa por examplo), então eu imagino que alguém bola algum modulo na OCA, mas se quiser fazer antes...

mileo committed:
[REF] fiscal.document.line fields_view_get -> _get_view

mileo commented:
/ocabot migration l10n_br_fiscal

mileo committed:
[REF] NCM / NBS fields_view_get -> UserError

REM fields_view_get and change UX to allow user to better understand
how to use the ibpt inquiry action.

mileo committed:
[FIX] test amount estimate tax to correpond core product amount

mileo commented:
Pessoal deixei o Menu assim: 

![image](https://user-images.githubusercontent.com/1975978/197645150-ec059268-4b6c-461c-8326-7ec779349cca.png)

Apenas uma linha de código, fácil de manter a diferença entre as versões. Se surgir um módulo na OCA pra isso repensamos.

mileo committed:
[FIX] certificate datetime test error

mileo committed:
[REF][l10n_br_fiscal] Menu max two levels

marcos-mendez commented:
> Pessoal na v16 pelo que eu vi não será possível mantermos essa estrutura de três níveis de menu:
> 
> ![image](https://user-images.githubusercontent.com/1975978/197440218-730f6025-79b4-4e23-9a72-21b0db225b8b.png)
> 
> Só esta sendo possível usar 2 níveis, se alguém tiver alguma sugestão de como organizar esses menus por favor comenta aqui.

No Caso Mileo o pessoal do /PMS resolveu isso de maneira bem interessante já desde a versão 14. Deixo aqui o link do RUNBOAT deles para vocês verem se é valido. 

https://runboat.odoo-community.org/webui/builds.html?repo=OCA/pms&target_branch=14.0

![image](https://user-images.githubusercontent.com/38361760/197777584-5f73abb0-4b16-4862-afde-b6bb50e0fdba.png)


Eu peguei um péssimo exemplo porque no caso o MENU lateral funciona como filtro, mas eu vi em algumas views tipo nos aplicativos que dá para ser como um SUBMenu do menu principal. 

Não sei se me explico

DiegoParadeda committed:
[FIX] expiration date field invalid date
